doc/coding_conventions/coding_conventions.txt

```diff
@@ -116,7 +116,11 @@ too much.
    /// Total number of elements
    unsigned nelement();
    \endcode
- \b Notes: (i) No underscore between the "n" and the container's name.
+ \b Notes: (i) No underscore between the "n" and the container's name;
+ You can then use the underscore for the name of the local variable
+ that stores the value, e.g.
+ \c
+ unsigned \c n_element \c = \c my_mesh_pt->nelement();
```

Comment:
Lowercase "y" in "you" on the second line?

Reply:
Yes sir! :) And thanks for adding the labels; I agree that they are really useful but I keep forgetting.